tools/bumper/parser_test.go

```diff
@@ -0,0 +1,185 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Testing internal parser", func() {
+
+	var tmpfile *os.File
+
+	CreateTempFileWithContent := func (content []byte) {
```

Comment:
Don't use mayus is not needed `createTempFileWithContent` is enough

Reply:
Done